engine/swarm/configs.md

```diff
@@ -612,3 +612,53 @@ configuration file.
 
 You have now updated your `nginx` service's configuration without the need to
 rebuild its image.
+
+### Example: Use templated config in Nginx service
+
+To create a configuration in which the content will be generated using a
+template engine, use the `--template-driver` parameter and specify the engine
+name as its argument. The template will be rendered when container is created.
+
+1.  Save the following into a new file `index.html`.
+
+    ```html
+    <html>
+      <head><title>Hello Docker</title></head>
+      <body>
+        <p>{% raw %}{{ env "HELLO" }}{% endraw %}! I'm service {% raw %}{{ .Service.Name }}{% endraw %}.</p>
+      </body>
+    </html>
+    ```
+
+2.  Save the `index.html` file as a swarm config named `homepage`. Provide
+    parameter `--template-driver` and specify `golang` as template engine.
+
+    ```bash
+    $ docker config create --template-driver golang homepage index.html
+    ```
+
+3.  Create a service that runs Nginx and has access to the environment variable
+    HELLO and to the config.
+
+    ```bash
+    $ docker service create \
+         --name nginx \
```

Comment:
Perhaps change this to `hellodocker` or (e.g.) `hello-template` to  make it more recognisable and to prevent it conflicting with other examples

```suggestion
         --name hello-template \
```